concepts/validator/index.md

```diff
@@ -45,38 +45,101 @@ Prerequisites for becoming a validator node operator are:
 The validator makes use of different accounts and private/public [key pairs](/glossary/#key-pair) for validator lifecycle management (registration, pausing, reactivation), validator identity, staking rewards, consensus participation and cryptographic security. 
 
 ### P2P node keys: autonityKeys
-The private/public key pair of the validator node.
+The private/public key pair of the validator node. The `autonitykeys` file contains the private keys for the transaction and consensus gossiping [communication layer](/concepts/architecture/#communication-layer). The keys are concatenated together to create a 128 character string:
 
-The private key is used:
+- the first 64 characters are the `node key`: used for transaction gossiping with other network peer nodes
```

Comment:
more than `transaction gossiping` I would say `transaction signing`. I saw this also in other parts of the PR

Reply:
Corrected to signing. (I'll look for instances elsewhere, too.)

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.